scripts/test-gobview.py

```diff
@@ -24,8 +24,8 @@ def cleanup(browser, thread):
 # serve GobView in different thread so it does not block the testing
 def serve():
   global p
-  goblint_http_path = '_build/default/gobview/goblint-http-server/goblint_http.exe'
-  p = subprocess.Popen(['./' + goblint_http_path,
+  goblint_http_path = './goblint-http'
+  p = subprocess.Popen([goblint_http_path,
                   '-with-goblint', '../analyzer/goblint',
                   '-goblint', '--set', 'files[+]', '"../analyzer/tests/regression/00-sanity/01-assert.c"'])
```

Comment:
These `-with-goblint`, `-goblint` and `--set` arguments are no longer needed? At least according to the updated inspecting documentation.